src/getPerspectiveCamera.es6.js

```diff
@@ -0,0 +1,20 @@
+import THREE from "three";
+import cycleCheck from "./cycleCheck.es6.js";
+
+function getPerspectiveCamera(cfg) {
+  if (typeof cfg === typeof undefined) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of undefined.](https://www.codacy.com/app/lagora-franck/proc-edu-ria/file/2572033293/issues/source?bid=3286583&fileBranchId=3298273#l5)